code/ship/ship.cpp

```diff
@@ -8336,8 +8346,8 @@ static void ship_dying_frame(object *objp, int ship_num)
 				}
 			}
 
-			// If this is a large ship with a propagating explosion, set it to blow up.
-			if ( ship_get_exp_propagates(shipp) )	{
```

Comment:
Wait.  So what happens to this function, and where food the code now use propagating explosions?

Reply:
It's still there (it just returns `sip->explosion_propagates`), and it's still used for the main thing everyone expects to be; enabling those smaller explosions before the big one.

Reply:
All right, the changes seem straight forward enough.  If you're sure it works, let's get it merged.